Cargo.toml

```diff
@@ -13,18 +13,23 @@ categories = ["rendering"]
 
 [dependencies]
 arrayref = "0.3"
-arrayvec = "0.5"
+arrayvec = { version = "0.5", default-features = false }
 bytemuck = "1.4"
 cfg-if = "1"
 png = { version = "0.16", optional = true }
 safe_arch = { version = "0.5.2", features = ["bytemuck"], optional = true }
+libm = { version = "0.2.1", optional = true }
 
 [features]
-default = ["simd", "png-format"]
+default = ["std", "simd", "png-format"]
+
+# Enables the use of the standard library. Deactivate this and activate the libm
+# feature to compile for targets that don't have std.
+std = []
 
 # Enables x86 SIMD instructions from SSE up to AVX2.
 # Has no effect on non-x86 targets. Present mainly for testing.
 simd = ["safe_arch"]
 
 # Allows loading and saving `Pixmap` as PNG.
-png-format = ["png"]
+png-format = ["std", "png"]
```

Comment:
The `png` crate requires `std`?

Reply:
I have not actually checked. I only saw file system usage in tiny-skia and noticed that at least that part needs to be std-only. But I could take a closer look.

Reply:
Then I guess we can move `fs` under it's own feature. Because tiny-skia already has two methods: one for encoding/decoding a raw data and one for fs.

Reply:
It does not seem like there is any no_std support in that crate. (Maybe that's something I can look into next xD)

Reply:
Ok, than we can leave it as std-only for now.